eng/code-quality-reports/src/test/java/com/azure/tools/checkstyle/checks/GoodLoggerCheckTests.java

```diff
@@ -0,0 +1,76 @@
+package com.azure.tools.checkstyle.checks;
```

Comment:
You're missing that MIT header on all your files.

Reply:
We are only check GoodLoggerCheck. This test data will not follow other rules but only point out the GoodLoggerCheck. It is a fake java file.

Reply:
These faked java files are under resource folder will be not be complied.

Reply:
This is the test class and should be checked for the license header

Reply:
My bad. This is not the fake java files. Interesting. The test still passed CI